cloudflare/utils.go

```diff
@@ -137,3 +138,17 @@ func stringFromBool(status bool) string {
 	}
 	return "off"
 }
+
+func getAccountID(d *schema.ResourceData, client *cloudflare.API) (string, error) {
```

Comment:
Bit of a nitpick but would this make more sense as `getAccountIDFromZoneID`? Explicitly calling out the source and expected returned data?

Reply:
I agree, `getAccountIDFromZoneID` is a better name for this.